hydroflow_datalog/src/lib.rs

```diff
@@ -352,4 +352,16 @@ mod tests {
             "#
         );
     }
+
+    #[test]
+    fn test_simple_filter() {
+        test_datalog_snapshot!(
+            r#"
+            .input input
+            .output out
+
+            out(x, y) :- input(x, y) | { x > y }.
```

Comment:
Once you add the code-gen logic for boolean expressions, we should also add runtime tests. Let's try to get those into this PR.